composer.json

```diff
@@ -19,11 +19,12 @@
         "drupal/core-dev": "^9.2",
         "drupal/entity_browser": "^2.6",
         "drupal/file_link": "^2.0.4",
+        "drupal/inline_entity_form": "^1.0-rc12",
         "drupal/typed_link": "^2.0",
         "drush/drush": "^10.3",
         "egulias/email-validator": "^2.1.22 || ^3.0",
         "openeuropa/code-review": "^1.6 || ^2.0",
-        "openeuropa/oe_content": "^2.8.0",
+        "openeuropa/oe_content": "3.x-dev",
```

Comment:
Given that this is require-dev, the main effect will be to determine which version will be tested.
There is no immediate effect on 3rd party projects, but it means future development on oe_starter_content can break oe_content 2.x compatibility.

I don't like the idea in oe_content that 2.x is no longer supported, while we have no stable release in 3.x. If this is the final word for oe_content, then the choice here seems ok.

An alternative might have been to try to support both versions temporarily, and do some conditional stuff to remove the patch from oe_content (*), and conditionally apply our own patches instead. I think I saw this somewhere, but I don't remember how it can be done.


_(*) Background: The composer.json of oe_content 2.x asks to apply a patch to inline_entity_form, which no longer works._

Reply:
oe_content 2.x is no longer actively maintained by core team, all the new work goes in 3.x .